include/stdexec/execution.hpp

```diff
@@ -3811,14 +3811,14 @@ namespace stdexec {
           template <class _Self, class _Receiver>
             using __operation_t =
               stdexec::__t<__operation<
-                stdexec::__id<__copy_cvref_t<_Self, _Sender>>,
+                stdexec::__cvref_id<_Self, _Sender>,
                 stdexec::__id<_Receiver>,
                 _Fun,
                 _Set>>;
           template <class _Self, class _Receiver>
             using __receiver_t =
               __receiver<
-                stdexec::__id<__copy_cvref_t<_Self, _Sender>>,
+                stdexec::__cvref_id<_Self, _Sender>,
```

Comment:
this isn't quite the same thing, since it won't preserve the ref-ness of `_Sender`, whereas before at least if `_Sender` were a ref and `_Self` were not, `__copy_cv_ref` would keep the ref (as far as I understand)

Reply:
This might not matter as the parent `__sender` type ends up `decay_t`-ing the `_Sender` already.

Reply:
Right. At this point, `_Sender` is not cvref-qualified.